packages/nest-commander/src/command-runner.service.ts

```diff
@@ -61,8 +61,10 @@ ${cliPluginError(
       this.commander.exitOverride(this.options.errorHandler);
     }
     if (!this.options.serviceErrorHandler) {
-      this.options.serviceErrorHandler = (err: Error) =>
+      this.options.serviceErrorHandler = async (err: Error) => {
         process.stderr.write(err.toString());
```

Comment:
If it bubbles the error, should printing it not be the responsibility of the caller? I.e. adding `console.error(err)` to the catch block in `main.ts`.